incubating/octopusdeploy-create-package/step.yaml

```diff
@@ -2,8 +2,8 @@ version: "1.0"
 kind: step-type
 metadata:
   name: octopusdeploy-create-package
-  version: 1.0.0
-  title: Create a package
+  version: 1.0.1
+  title: Create a package in Octopus Deploy
```

Comment:
The package isn't really created "in" Octopus Deploy... What about "Create Octopus Deploy package"?

Reply:
Done